src/common/offline_transformations/src/pruning/shrink_weights.cpp

```diff
@@ -70,6 +70,103 @@ static bool maybe_adopt_reshape_node(std::shared_ptr<ov::Node> reshape, ngraph::
     return true;
 }
 
+static bool handle_variadic_split(const std::shared_ptr<ov::Node>& split) {
+    const auto axis_node = ngraph::as_type<ngraph::opset6::Constant>(split->get_input_node_ptr(1));
+    if (!axis_node)
+        return false;
+
+    const auto& input_shape = split->get_input_partial_shape(0);
+    if (input_shape.rank().is_dynamic())
+        return false;
+
+    auto axis = axis_node->cast_vector<int64_t>()[0];
+    if (axis < 0)
+        axis += input_shape.size();
+
+    if (input_shape[axis].is_dynamic())
+        return false;
+
+    const auto split_lengths_node = ngraph::as_type<ngraph::opset6::Constant>(split->get_input_node_ptr(2));
+    if (!split_lengths_node)
+        return false;
+    auto split_lengths = split_lengths_node->cast_vector<int64_t>();
```

Comment:
Please use `const` keywork if value of variable shouldn't be modified
```suggestion
    const auto split_lengths = split_lengths_node->cast_vector<int64_t>();
```

Reply:
done